django_comments/migrations/0001_initial.py

```diff
@@ -8,7 +8,6 @@
 class Migration(migrations.Migration):
 
     dependencies = [
-        ('sites', '0001_initial'),
```

Comment:
I think an extension of `ForeignKey` would be needed.  One that could handle 'sites' being turned on/off or possibly give a descriptive warning if 'sites' is turned on/off without changing the database properly.